cl/lib/email_backends.py

```diff
@@ -96,7 +211,10 @@ def send_messages(
             if final_recipient_list:
                 # Update message with the final recipient list
                 email.to = final_recipient_list
+                # check the emergency brake before sending an email
```

Comment:
We should just move up this comment before `check_emergency_brake(r)`.

Reply:
I appreciate you bringing that to my attention. I'm changing this.